boards/xtensa/intel_adsp_cavs25/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+# SPDX-License-Identifier: Apache-2.0
+
+find_program(
+  RIMAGE
+  rimage
+  )
+
+if(CONFIG_BOARD_INTEL_ADSP_CAVS25 OR CONFIG_BOARD_INTEL_ADSP_CAVS25_TGPH)
```

Comment:
this is a common step for all intel adsp platforms, I think we can put it in "soc/xtensa/intel_adsp/common/CMakeLists.txt".

Reply:
yes, good point, will try to do it this way :), thanks!

Reply:
Yes, I tried to put some common part in to "soc/xtensa/intel_adsp/common/CMakeLists.txt". For the "key" using to signed rimage I still put them in their own board.cmake.